### PR TITLE
Typos, text and code fix in LDA tutorial

### PR DESCRIPTION
**Pull request by davebulaval**
This PR fixes the following in all three files format:

- unused import (`os.path` and `io`)
- Fix some dead code
- Propose some text improvements

Moreover, the following link is broken [http://rare-technologies.com/lda-training-tips/](http://rare-technologies.com/lda-training-tips/), and I was not able to find the new URL.

image

### CODE REVIEW

File: docs/src/auto_examples/tutorials/run_lda.py [hunk 1]
```diff
@@ -220,7 +219,7 @@ def extract_documents(url='https://cs.nyu.edu/~roweis/data/nips12raw_str602.tgz'
 #
 # I suggest the following way to choose iterations and passes. First, enable
 # logging (as described in many Gensim tutorials), and set ``eval_every = 1``
-# in ``LdaModel``. When training the model look for a line in the log that
+# in ``LdaModel``. Then, when training the model, look a line in the log that
```
Comment (piskvorky) [inline docs/src/auto_examples/tutorials/run_lda.py:222]: -1: the new version doesn't make sense.

**Review by piskvorky — CHANGES_REQUESTED** [review]

File: docs/src/auto_examples/tutorials/run_lda.py (hunk 1 above)
Comment (davebulaval) [reply docs/src/auto_examples/tutorials/run_lda.py:222]: I agree that the for is changing the sense.

I propose:

`Then, when training the model, look for a line in the log that`

**Comment by piskvorky** [comment]
Yeah, that phrase `as described in many Gensim tutorials` in unnecessarily cryptic.

Can you include this instead?

```python
logging.basicConfig(level=logging.INFO, format='PID:%(process)d:%(threadName)s - %(asctime)s - %(levelname)s - %(filename)s:%(lineno)s - %(message)s')
```

Thanks!

File: docs/src/auto_examples/tutorials/run_lda.py (hunk 1 above)
Comment (davebulaval) [reply docs/src/auto_examples/tutorials/run_lda.py:222]: 97cce0e8

**Review by piskvorky — APPROVED** [review]

**Comment by piskvorky** [comment]
LGTM, thanks! The failing wheels seem unrelated, @mpenkov is working on those.

**Comment by davebulaval** [comment]
@piskvorky glad to help!

**Pull request merged**